test/bithelp-test.js

```diff
@@ -0,0 +1,71 @@
+'use strict';
+
+const expect = require('chai').expect;
+const bitmapFileHelper = require ('../lib/bitmap-helper.js');
+
+describe('testing color-transform.js', function(){
+  describe(' Bitmap constructor test', function(){
+    it('should return number of colors', function(done){
+      bitmapFileHelper.read(`${__dirname}/../img/palette-bitmap.bmp`, function(err, data){
+        expect(data.numOfColors).to.equal(256);
+        done();
+      });
+    });
+
+  });
+});
+describe('testing color-transform.js', function(){
+  describe(' Bitmap constructor test', function(){
+    it('should return pixel array offset', function(done){
+      bitmapFileHelper.read(`${__dirname}/../img/palette-bitmap.bmp`, function(err, data){
+        expect(data.pixelArrayOffset).to.equal(1078);
+        done();
+      });
+    });
+
+  });
+});
+describe('testing color-transform.js', function(){
+  describe(' Bitmap constructor test', function(){
+    it('should size', function(done){
+      bitmapFileHelper.read(`${__dirname}/../img/palette-bitmap.bmp`, function(err, data){
+        expect(data.size).to.equal(11078);
+        done();
+      });
+    });
+
+  });
+});
+describe('testing color-transform.js', function(){
+  describe(' Bitmap constructor test', function(){
+    it('should return the correct type', function(done){
+      bitmapFileHelper.read(`${__dirname}/../img/palette-bitmap.bmp`, function(err, data){
+        expect(data.type).to.equal('BM');
+        done();
+      });
+    });
+
+  });
+});
```

Comment:
@jtwalters25 @zcrumbo @sueanyv tests look good - nice job checking metadata about the bitmap, including the size and type